setup.py

```diff
@@ -1,8 +1,11 @@
-# Copyright 2015 Knowledge Economy Developments Ltd
+# Copyright 2017 Henry Gomersall and Frederik Beaujean
```

Comment:
Should this maybe be something generic like "The PyFFTW Contributors"?  I know a few others have made contributions to this file in the past as well.